src/config.rs

```diff
@@ -133,12 +136,22 @@ pub struct Config {
     pub metrics_server: Server,
     pub database: Database,
     pub secrets: Secrets,
+    pub cassandra: Cassandra,
     pub log: LogConfig,
     pub pool_config: PoolConfig,
     #[cfg(feature = "mtls")]
     pub certs: Certs,
 }
 
+#[derive(Deserialize, Debug)]
+pub struct Cassandra {
+    pub known_nodes: String,
```

Comment:
Is this supposed to be a `HashSet` or a `Vec`?

Reply:
Fixed, thanks